### PR TITLE
Cassandra related files for ich_segmentation

### PR DESCRIPTION
**Pull request by mdreyesm**
Files required to configure and use Cassandra for ich_segmentation use case.

### CODE REVIEW

File: healthcare/ich_segmentation/cassandra/cassandra-upstream-chart/files/conf/jmxremote.password [hunk 1]
```diff
@@ -0,0 +1 @@
+jmx_user cassandra
```
Comment (unrahul) [inline healthcare/ich_segmentation/cassandra/cassandra-upstream-chart/files/conf/jmxremote.password:1]: is this a  hardcoded password? if so please remove it

File: healthcare/ich_segmentation/cassandra/cql/cqlshrc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[authentication]
+username = user
+password = password
```
Comment (unrahul) [inline healthcare/ich_segmentation/cassandra/cql/cqlshrc:3]: the same here, please remove hardcoded option, we dont want our users to use these files as is, with hardcoded authentication info.

File: healthcare/ich_segmentation/cassandra/cassandra-upstream-chart/files/conf/cassandra-env.sh [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+  JVM_OPTS="$JVM_OPTS -Dcom.sun.management.jmxremote.authenticate=true"
+
+  # jmx ssl options
+  JVM_OPTS="$JVM_OPTS -Dcom.sun.management.jmxremote.ssl=true"
+  JVM_OPTS="$JVM_OPTS -Dcom.sun.management.jmxremote.ssl.need.client.auth=false"
+  JVM_OPTS="$JVM_OPTS -Dcom.sun.management.jmxremote.registry.ssl=true"
+  #JVM_OPTS="$JVM_OPTS -Dcom.sun.management.jmxremote.ssl.enabled.protocols=<enabled-protocols>"
+  #JVM_OPTS="$JVM_OPTS -Dcom.sun.management.jmxremote.ssl.enabled.cipher.suites=<enabled-cipher-suites>"
+  JVM_OPTS="$JVM_OPTS -Djavax.net.ssl.keyStore=/etc/cassandra/keystore-decoded"
+  JVM_OPTS="$JVM_OPTS -Djavax.net.ssl.keyStorePassword=cassandra"
```
Comment (unrahul) [inline healthcare/ich_segmentation/cassandra/cassandra-upstream-chart/files/conf/cassandra-env.sh:279]: no hardcoded pass please

File: healthcare/ich_segmentation/cassandra/cql/nodetool-ssl.properties [hunk 4]
```diff
@@ -0,0 +1,7 @@
+-Dcom.sun.management.jmxremote.ssl=true
+-Dcom.sun.management.jmxremote.ssl.need.client.auth=false
+-Dcom.sun.management.jmxremote.registry.ssl=true  
+-Djavax.net.ssl.keyStore=/path/to/keystore/file/.keystore
+-Djavax.net.ssl.keyStorePassword=cassandra
```
Comment (unrahul) [inline healthcare/ich_segmentation/cassandra/cql/nodetool-ssl.properties:5]: remove hardcoded pass

File: healthcare/ich_segmentation/cassandra/cassandra-upstream-chart/files/conf/cassandra-env.sh (hunk 3 above)
Comment (unrahul) [inline healthcare/ich_segmentation/cassandra/cassandra-upstream-chart/files/conf/cassandra-env.sh:279]: remove hardcoded pass

File: healthcare/ich_segmentation/cassandra/cassandra-upstream-chart/files/conf/jmxremote.password (hunk 1 above)
Comment (mdreyesm) [reply healthcare/ich_segmentation/cassandra/cassandra-upstream-chart/files/conf/jmxremote.password]: Those are example files, I added .example extension on those files and I changed the passwor to something like "<PASSWORD_FOR_DB>".

File: healthcare/ich_segmentation/cassandra/cql/cqlshrc (hunk 2 above)
Comment (mdreyesm) [reply healthcare/ich_segmentation/cassandra/cql/cqlshrc]: Same case as jmx examples.

File: healthcare/ich_segmentation/cassandra/cassandra-upstream-chart/files/conf/cassandra-env.sh (hunk 3 above)
Comment (mdreyesm) [reply healthcare/ich_segmentation/cassandra/cassandra-upstream-chart/files/conf/cassandra-env.sh]: Same case as jmx examples.

File: healthcare/ich_segmentation/cassandra/cql/nodetool-ssl.properties (hunk 4 above)
Comment (mdreyesm) [reply healthcare/ich_segmentation/cassandra/cql/nodetool-ssl.properties]: Same case as jmx examples.

File: healthcare/ich_segmentation/cassandra/cassandra-upstream-chart/files/conf/cassandra-env.sh (hunk 3 above)
Comment (mdreyesm) [reply healthcare/ich_segmentation/cassandra/cassandra-upstream-chart/files/conf/cassandra-env.sh]: Same case as jmx examples.

**Pull request merged**